zep_python/memory/client.py

```diff
@@ -829,16 +830,19 @@ def synthesize_question(self, session_id: str, last_n: int = 3) -> str:
         APIError
             If the API response format is unexpected.
         """
+        print("session_id inside synthesize_question", session_id)
         if session_id is None or session_id.strip() == "":
             raise ValueError("session_id must be provided")
-
+        print("Inside synthesize_question")
         params = {"lastNMessages": last_n}
         response = self.client.get(
             f"/sessions/{session_id}/synthesize_question", params=params
         )
 
         handle_response(response)
 
+        print("response.json()", response.json())
```

Comment:
suggest using a debugger and not adding debug code to SDK. Less risk of leaving debug prints in merges.